src/OSPSuite.Presentation/Presenters/Charts/CurveMultiItemEditorPresenter.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Drawing;
+using OSPSuite.Core.Chart;
+using OSPSuite.Presentation.Views.Charts;
+
+namespace OSPSuite.Presentation.Presenters.Charts
+{
+   public enum YesNoValues
+   {
+      Yes,
+      No
+   }
+
+   public interface ICurveMultiItemEditorPresenter : IDisposablePresenter, IPresenter<ICurveMultiItemEditorView>
+   {
+      void Show();
+      bool Canceled();
+      SelectedCurveValues GetSelectedValues();
+   }
+
+   public class SelectedCurveValues
+   {
+      public Color? Color { get; set; }
+      public string Style { get; set; }
+      public string Symbol { get; set; }
```

Comment:
Why is everything designed as string?we have specific enums etc..this is very suspicious

Reply:
@msevestre let us briefly talk about this...I used to have them as nullable enum, but was not very fond of that behaviour either.